modules/gapi/misc/python/test/test_gapi_core.py

```diff
@@ -20,8 +20,8 @@ class gapi_core_test(NewOpenCVTests):
     def test_add(self):
         # TODO: Extend to use any type and size here
         sz = (1280, 720)
-        in1 = np.random.randint(0, 100, sz).astype(np.uint8)
-        in2 = np.random.randint(0, 100, sz).astype(np.uint8)
+        in1 = np.random.randint(0, 100, sz)
+        in2 = np.random.randint(0, 100, sz)
```

Comment:
please add check that `actual.dtype` matches `expected.dtype`. And keep at least one test with `.astype(np.uint8)` so we can validate U8 as well

Reply:
Fixed